src/models/ClassPotentialR2HDM.cpp

```diff
@@ -228,6 +228,11 @@ void Class_Potential_R2HDM::set_gen(const std::vector<double> &par)
   C_SinBetaSquared = TanBeta * TanBeta * C_CosBetaSquared;
   C_SinBeta        = sqrt(C_SinBetaSquared);
 
+  if (TanBeta < 0) // for beta in 4th quadrant
+  {
+    C_SinBeta *= -1;
+  }
+
```

Comment:
What happens in the case cos(beta)< 0 ? This would lead to a misassignment of signs if cos(beta) < 0 but sin(beta) > 0

Reply:
We cannot access the region with cos(beta) < 0, as far as I see it. From TanBeta we only get beta in [-Pi/2, Pi/2] by using [`std::atan`](https://en.cppreference.com/w/cpp/numeric/math/atan). With the above fix, we now also correctly reproduce the region where sin(beta) < 0. The cosine is cos(beta) > 0 for beta in [-Pi/2, Pi/2].

Reply:
Alright. Can you add a unit test for this case?

Reply:
I've added a unit test that checks the sign of `C_CosBeta` and `C_SinBeta` in 32fe610 for the `example_point_R2HDM`. Shall I add a second example point with negative `TanBeta` to the check?

Reply:
Done in feb36d4.